db/migrate/20220102154755_add_basic_info_to_users.rb

```diff
@@ -0,0 +1,8 @@
+class AddBasicInfoToUsers < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :name, :string, {null: false, default: ""}
+    add_column :users, :birthday, :date
+    add_column :users, :sex, :integer, {default: 0} #User.sex_type.none
+    add_column :users, :school, :string, {null: false, default: ""}
```

Comment:
ruby だと引数末尾の Hash は `{}` を省略できるので以下の様に書けます。
`null` 禁止や `default` 設定されている点はさすがです 👍 

```suggestion
    add_column :users, :name, :string, null: false, default: ""
    add_column :users, :birthday, :date
    add_column :users, :sex, :integer, default: 0 # User.sex_type.none
    add_column :users, :school, :string, null: false, default: ""
```

Reply:
Ruby の書き方は色々合って迷うと思うので、 `rubocop` を導入しておくと良いと思います。
エディタによってはショートカットキーで RuboCop を実行して自動修正までしてくれます。

また、ソーシャルPLUS では CI で RuboCop によるチェックを実行しています。

[RuboCopのインストール手順と具体的な使い方 \| Enjoy IT Life](https://nishinatoshiharu.com/insatall-rubocop/#RuboCop)

Reply:
マニュアルを参照したところ、次のような引数であったため、4つの引数でないといけないと解釈していました。
> add_column(テーブル名 カラム名, タイプ, オプション引数)

> ruby だと引数末尾の Hash は `{}` を省略できる
覚えておきます。

後ほどRuboCop を導入いたします。